server/src/main/java/org/apache/cassandra/sidecar/server/MainModule.java

```diff
@@ -741,7 +741,8 @@ private static InstanceMetadata buildInstanceMetadata(Vertx vertx,
                                        .password(cassandraInstance.jmxRolePassword())
                                        .enableSsl(cassandraInstance.jmxSslEnabled())
                                        .connectionMaxRetries(jmxConfiguration.maxRetries())
-                                       .connectionRetryDelayMillis(jmxConfiguration.retryDelayMillis())
+                                       // TODO: connectionRetryDelayMillis should be a MillisecondBoundConfiguration
```

Comment:
Why postponing it?

Reply:
mostly because it'd require a refactor of the JmxClient class which lives in a different subproject. I will give it a go though, I think it makes sense to do the change here

Reply:
done